scripts/kibana_jsonc_migration.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+var Fs = require('fs');
```

Comment:
Is this something we want to keep here on this PR?

Reply:
Not long term. I want to leave it in for now though and remove it just before merge so that I can remigrate all the manifests later when merging main.